charts/juicefs-csi-driver/templates/daemonset.yaml

```diff
@@ -45,7 +45,7 @@ spec:
         {{- end }}
         env:
         - name: CSI_ENDPOINT
-          value: unix:/csi/csi.sock
+          value: unix:/csi/socket
```

Comment:
Why change `csi.sock` to `socket`? The `csi.sock` is a typical one. 

After turning `csi-plugins` to `plugings`, `csi.sock` still doesn't work?